package.json

```
@@ -14,6 +14,7 @@
     "build": "rspack build",
     "preview": "NODE_ENV=production node ./server.js",
     "ssr": "node build/ssr.js",
+    "lit-analyzer": "npx @jackolope/lit-analyzer --rules.no-incompatible-type-binding off",
```

You shouldn't need to use npx in the scripts

```suggestion
    "lit-analyzer": "@jackolope/lit-analyzer --rules.no-incompatible-type-binding off",
```